docker/Dockerfile

```diff
@@ -1,19 +1,18 @@
 FROM golang:1.20-alpine AS builder
-ARG SVC
 ARG GOARCH
 ARG GOARM
 
 WORKDIR /go/src/github.com/leoparente/opentelemetry-infinity
```

Comment:
we need a additional step that downloads latest otelcol-contrib binary and put it into runner/ folder, otherwise it will not build! https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.75.0/otelcol-contrib_0.75.0_linux_amd64.tar.gz

 I was not able to add directly the artifact because it is too big to be added to code directly.

Reply:
yeah I know! I was getting error during build